_docs/cardano/2017-01-04-transactions.md

```diff
@@ -3,41 +3,113 @@ layout: default
+/                       |           |                                              |
+------------------------+           +----------------------------------------------+                                             
+~~~
+
+Transaction `f9bcbe752aee4512..` informs us that:
+
+1. we want to send 100 ADA to the node with address `88ca7f79..`;
+2. this money corresponds to `0`-th output of the previous transaction with an id `891971a4c..`.
+
+So in this case we know that 100 ADA transfers from the current node (who generates transaction `f9bcbe752aee4512..`)
+to the node `88ca7f79..`, and this money was received from the node `a00e4bb2..`.
+
+## Verification
+
+As said above, transaction's output become an input of the other transaction. In this case we treat such an
+output as _spent transaction output_. Thus, an output `Out0` of the transaction `891971a4cc31e32..` is spent output
+because it already became an input of the transaction `f9bcbe752aee4512..`
+
+But such a spending doesn't occur immediately, so an output which _isn't yet_ an input of another transaction
+called _unspent transaction output_, or "utxo". Only utxo can be used as an input for the other transaction,
+to prevent [double-spending](https://en.bitcoin.it/wiki/Double-spending).
+
+So every node in the network not just accepts transactions, but [verify](https://github.com/input-output-hk/cardano-sl/blob/63adb31e813e21ec9da21cfa69984840308bbfa2/src/Pos/Types/Tx.hs#L91) them. To do it,
+every node have to keep track of utxo, it allows to validate that inputs in a published
+transaction are indeed the unspent outputs.
 
 ## Proofs of Transaction Legitimacy
 
 Each transaction in Cardano SL is accompanied by a proof (also called a
-witness) that this transaction is legitimate. Those proofs are stored in
-the blockchain and anybody can see, inspect and independently verify
-them. However, not everybody needs to do that – for instance, if you are
-an SPV node, you are trusting other nodes to do the verification, and
-thus you would rather not have those proofs sent to you with every
-transaction, as they are useless to you. Moreover, even if you are a
-full node, you might want to delete old proofs after some time in order
-to save space.
+[witness](https://github.com/input-output-hk/cardano-sl/blob/63adb31e813e21ec9da21cfa69984840308bbfa2/src/Pos/Types/Types.hs#L93)) that this transaction is legitimate. Witness for transaction `TxN` consists of the witnesses of
+all `TxN`'s inputs. Witnesses are stored in the blockchain and anybody can see, inspect 
+and independently verify them. But after some time node can delete old proofs in order to save space.
 
 The technique of storing transactions separately from their proofs is
-called “segregated witness” (you may have heard of it being recently
+called "segregated witness" (you may have heard of it being recently
 implemented in Bitcoin). Under this scheme, transactions and proofs are
```

Comment:
@denisshevchenko Why would you change right quotes to wrong ones? (I don't care much, but still curious.)